src/it/resources/application.conf

```diff
@@ -0,0 +1,8 @@
+gargoyle {
+    sts {
+        keycloak {
+            token.url = "http://127.0.0.1:8080/auth/realms/auth-gargoyle/protocol/openid-connect/token"
```

Comment:
this default value can go to the reference.conf

Reply:
done